mne/tests/test_surface.py

```diff
@@ -95,20 +95,27 @@ def test_make_morph_maps():
     for subject in ('sample', 'sample_ds', 'fsaverage_ds'):
         os.mkdir(op.join(tempdir, subject))
         os.mkdir(op.join(tempdir, subject, 'surf'))
+        regs = ('reg', 'left_right') if subject == 'fsaverage_ds' else ('reg',)
```

Comment:
what is this conditional for `fsaverage_ds`?

Reply:
(`fsaverage_ds` shouldn't be treated any differently within MNE from any other subject -- it just so happens that we've downsampled it for faster processing)

Reply:
This is just copying files needed for a test so that the test does not write into the testing data folder.